frontend/src/helpers/questionnaire.helper.ts

```diff
@@ -0,0 +1,15 @@
+import {IQuestionnaire} from "../models/forms/Questionnaires/types";
+
+export default function (questionnaireId: string, questionId: string, questionnaires: IQuestionnaire[]){
+    for (const questionnaire of questionnaires) {
```

Comment:
I'm not an expert, but maybe something like this would do the trick?
```
questionaires.filter(q => q.id !== questionnaireId).flatMap(q => q.questions).some(question => question.id === question.id)
```